services/storage-calculator/create_jwt.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+
+import os
+import jwt
+
+payload = {'role': 'admin', 'iss': 'storage-calculator',
```

Comment:
The `sh` script used `iss`/`sub`: `auto-idler` here, I changed it to `storage-calculator`. Not sure if that will break.. does the API do any checks on this field?

Reply:
no, should be fine